### ISSUE TITLE
some questions about dataset process

### ISSUE THREAD

**Issue by WHU-YH-jx** [YaoleiQi/DSCNet#11]
Congratulations! Really nice work!
I have some questions about the ROAD dataset.
"All data enhancements are performed simultaneously during the training process. The random crop size of 256 × 256 is used as a fixed size to feed into the model training."
First, in the training process, the size of images is 256 × 256, the method of resizing is nearest-neighbor interpolation or linear interpolation ?
Second, what about the test process, is it also 256×256?
Third, your final result is a range, such as 78.21±5.77, what's the meaning of it and how do you get this result?

Thanks very much!

**Comment by YaoleiQi** [comment]
Thank you for your attention!
1. We did not perform any sampling operations on the original image and used the cropping method for the both training and testing processes.
2. The result is the mean and variance, which are obtained using Dice's operations

**Comment by WHU-YH-jx** [comment]
Thanks for reply!
So for the road dataset, random cropping was used in both training and testing?

**Comment by YaoleiQi** [comment]
Yes, training uses random cropping and testing uses regular cropping

**Comment by WHU-YH-jx** [comment]
Thanks!
So could you show the details of regular cropping in test stage for dataset Road. It is important for evaluating models.



---Original---
From: ***@***.***&gt;
Date: Fri, Sep 8, 2023 14:42 PM
To: ***@***.***&gt;;
Cc: "Ordinary ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [YaoleiQi/DSCNet] some questions about dataset process (Issue#11)




 
Yes, training uses random cropping and testing uses regular cropping
